### PR TITLE
Update Remove-NetIPAddress.md

### PR DESCRIPTION
**Pull request by JulianTurner**
changed the description of the comment because it was missleading and incomplete

### CODE REVIEW

**Comment by e0i** [comment]
@JulianTurner Please change the target branch of the PR from `live` to `master` to continue with the review. Thank you.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/nettcpip/Remove-NetIPAddress.md [hunk 1]
```diff
@@ -48,16 +48,16 @@ If you do not specify an IPv4 or IPv6 address, the cmdlet will remove all IP add
...
 ## EXAMPLES
 
-### Example 1: Remove an IP address
+### Example 1: Add and Remove an IP address
 ```
+This command adds a new IP address to the controller with the interfaceindex 12
 PS C:\>New-NetIPAddress -InterfaceIndex 12 -IPAddress 192.168.0.1
 
 
 This command removes the IP address created by the previous command.
```
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md]:
```suggestion
```

File: docset/windows/nettcpip/Remove-NetIPAddress.md [hunk 2]
```diff
@@ -48,16 +48,16 @@ If you do not specify an IPv4 or IPv6 address, the cmdlet will remove all IP add
...
 PS C:\>Remove-NetIPAddress -IPAddress 192.168.0.1
 ```
```
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md]:
```suggestion

The first command adds a new IPv4 address to the network interface at index 12. The second command removes the IPv4 address created by the previous command.
```

File: docset/windows/nettcpip/Remove-NetIPAddress.md [hunk 3]
```diff
@@ -48,16 +48,16 @@ If you do not specify an IPv4 or IPv6 address, the cmdlet will remove all IP add
 
 ## EXAMPLES
 
-### Example 1: Remove an IP address
+### Example 1: Add and Remove an IP address
 ```
+This command adds a new IP address to the controller with the interfaceindex 12
```
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md]:
```suggestion
```

**Comment by dariomws** [comment]
The same applies to New-NetIPAddress.

**Comment by get-itips** [comment]
Agree with @dariomws 
There is no need to put the New- cmdlet in the example, I think that was already redundant.
I don't really remember other Remove- articles that include in the example the New- or Add-

**Comment by e0i** [comment]
Duplicate of #1772.

**Pull request closed without merging**